app/controllers/ctc/questions/stimulus_received_controller.rb

```diff
@@ -7,13 +7,13 @@ class StimulusReceivedController < QuestionsController
 
       def self.show?(intake)
         return false if intake.eip1_amount_received.nil? || intake.eip2_amount_received.nil?
-
-        intake.tax_return(2020).outstanding_recovery_rebate_amount <= 0
+        intake.tax_return(2020).outstanding_recovery_rebate_credit.zero?
```

Comment:
The logic for stimulus received displaying was flip flopped. "Stimulus Received" text says:
          title: "Based on your info, it looks like you’ve received your full stimulus payments."

It should display only when the outstanding amount is determined to be 0.